src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/PackagesPreferencesPane.java

```diff
@@ -70,6 +74,11 @@ public PackagesPreferencesPane(PreferencesDialogResources res,
       VerticalPanel development = new VerticalPanel();
     
       management.add(headerLabel("Package Management"));
+
+      infoBar_ = new InfoBar(InfoBar.WARNING);
+      infoBar_.setText("CRAN repos were modified outside package preferences.");
```

Comment:
nit: can we use 'repositories' instead of 'repos' here?

Reply:
It might be worth stating more explicitly something like:

```
The active repositories have been overridden by a startup profile.
```

and, if we have some documentation on this somewhere, provide a help link to that article explaining the situation.